README.md

```diff
@@ -1,38 +1,51 @@
+  <figcaption>Figure: Overview of KinBADL pipeline</figcaption>
+</figure>
 
-## Dependency
-* Tested with [Python 3.7](https://www.python.org/downloads/) (Pandas) 
+## Dependencies
+* [Python 3.7](https://www.python.org/downloads/) (Pandas) 
 * [Rosetta](https://www.rosettacommons.org/software/license-and-download) Software Suite 
 * [PyRosetta](http://www.pyrosetta.org/) Software Suite
 * [OpenEye](https://www.eyesopen.com/) Software Suite
 * [EMBOSS](http://emboss.open-bio.org/html/use/ch02s07.html) Software Suite
 
-Initial folder should contain these three files and similar naming convention,
-
 ```
 2W1C_A_L0C
 ├── 2W1C_A.fasta
 ├── 2W1C_A_L0C.pdb
 └── L0C.smi
 ```
-Here are the STEPS and SCRIPTS used for the comparative modeling pipeline approach,
-
-## 1. Conformer generation and ligand alignment (modeling_script.py)
-* First, the maximum number of conformers will be generated for a given SMILES using OMEGA. The output will be a single SDF file with the maximum number of conformers.
-* Second, the single SDF file will be used for the alignment of a query (Molecule of interest) and database (in-house active kinase ligand template library) molecules using ROCS. The output will be each template aligned query molecule.
-* Third, combining the report files of each template aligned query molecule into a single report file.
-* Fourth, based on the single report file, the top 100 conformers for the given SMILES are selected.
-* Fifth, SDF to PARAMS will be generated using 100 conformers for Rosetta Minimization step. 
+
+
+The initial folder should contain three files and following the naming convention
+- ```2W1C_A_L0C.fasta``` is the FASTA protein file for the kinase 2W1C
+- ```2W1C_A_L0C.pdb``` is the PDB structure of 2W1C in complex with the L0C ligand
+- ```L0C.smi``` is the SMILES string for the L0C ligand
+
+
+## 1. Conformer generation and ligand alignment with ```modeling_script.py```
+* This script generates conformers, aligns conformers with a template, and selects the top 100 conformers for Rosetta minimization.
+
+An example command is below:
 ```
-python /path/to/Rosetta_Kinase_CM/modeling_script.py -f /path/to/2W1C_A_L0C -omega /path/to/omega2 -rocs /path/to/rocs -temp_lig /path/to/Rosetta_Kinase_CM/template_ligand_library -mol2params /path/to/Rosetta/main/source/scripts/python/public/generic_potential/mol2genparams.py -convert /path/to/Rosetta_Kinase_CM/convert.py
+/work/07424/gabeong/stampede2/anaconda3/bin/python /work/07424/gabeong/stampede2/rosetta_cm/Rosetta_Kinase_CM/modeling_script.py -f /work/07424/gabeong/stampede2/2W1C_A_L0C/ -omega /work/07424/gabeong/stampede2/openeye/bin/omega2 -rocs /work/07424/gabeong/stampede2/openeye/bin/rocs -temp_lig /work/07424/gabeong/stampede2/rosetta_cm/Rosetta_Kinase_CM/template_ligand_library -mol2params /work/07424/gabeong/stampede2/Rosetta/main/source/scripts/python/public/generic_potential/mol2genparams.py -convert /work/07424/gabeong/stampede2/rosetta_cm/Rosetta_Kinase_CM/convert.py
```

Comment:
@wgabrielong Do you want to specify the real path? I feel it is better to mention "/path/to/" symbol.

Reply:
@pkiruba do you think it is a problem? If not I think it makes things more clear, at least for somebody like me who is not familiar with working in Linux.

Reply:
Updated README